.github/workflows/label_pr.yml

```diff
@@ -1,13 +0,0 @@
-name: "Pull Request Labeler"
-on:
-  pull_request_target:
-    types:
-    - opened
-
-jobs:
-  triage:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/labeler@main
-      with:
-        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I'm not necessarily opposed to removing this, but is the problem just that the token isn't updated?  If so maybe we should fix that instead?  Or are you actively opposed to using the labeler at all?

Reply:
We currently don't have any actions for the labeler right now -- no configuration file that specifies what changed to look for a what labels to apply, so I'm sure what the point of having the workflow is right now.